x/mint/client/testutil/cli_test.go

```diff
@@ -1,5 +1,3 @@
-// +build norace
```

Comment:
I believe we need those for CLI tests, no? Or else there might be some race conditions

Reply:
I was testing integration locally. Forgot to add them back.